x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -236,7 +236,14 @@ export const enrichExceptionItemsWithOS = (
 export const retrieveAlertOsTypes = (alertData?: AlertData): OsTypeArray => {
   const osDefaults: OsTypeArray = ['windows', 'macos'];
   if (alertData != null) {
-    const os = alertData.host && alertData.host.os && alertData.host.os.family;
+    const osTypeBasedOnAgentType = (data: AlertData) => {
```

Comment:
(optional) suggestion:
this function seems pure and only used once. 
if you need to keep it, I would move it outside of the `retrieveAlertOsTypes()` so that you don't have to defined it on very call. 
Or: remove it, since you only seem to use it once.

Reply:
done